compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SymbolEnter.java

```diff
@@ -1159,16 +1159,21 @@ public void initPredeclaredModules(Map<Name, BPackageSymbol> predeclaredModules,
                         new Name(compUnits.get(index++).name)));
                 entry = this.env.scope.lookup(alias);
             }
+            if (!(entry.symbol != null && entry.symbol instanceof BPackageSymbol)) {
```

Comment:
Cannot we use `entry.symbol.kind` instead of `instanceof`?

Reply:
I will check it

Reply:
`entry.symbol.kind = null`, when the class cast exception occurred.

Reply:
![boolean_debug](https://user-images.githubusercontent.com/47134066/172543204-eb07cfc5-88e2-475a-9c2c-bf116753c048.png)

Reply:
If the symbol.kind is null, we need to check why it is null. IMO, it cannot be null. @MaryamZi WDYT?

Reply:
I agree with you.